pkgs/development/python-modules/pwinput/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  lib,
+}:
+buildPythonPackage rec {
+  pname = "pwinput";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "pwinput";
+    hash = "sha256-yhqL0G4ohy11Hb1BMthjcSfCW0COo6NJN3MUpUkUJvM=";
+  };
+
+  build-system = [ setuptools ];
+
+  # Requires graphical environment to use pyautogui
+  doCheck = false;
+
```

Comment:
Please add `pythonImportsCheck`.

Reply:
`pwinput` doesn't use any third-party `modules`, but I've added the native modules it imports.